test/test_torch.py

```diff
@@ -14662,6 +14662,263 @@ def test_ones_like_multiple_device(self, devices):
         self.assertEqual(output, expected)
 
 
+# Tests ops and indexing to ensure they return views (and new tensors) as
```

Comment:
Ho that's a very good point. We want that `base is not other` right (maybe add it to the `is_view_of()` function)? If I do `b = a.view_as(a)`, I want `b` to be a brand new Tensor.
But that can be added later.